internal/vds/vds.go

```diff
@@ -55,6 +55,12 @@ type Metadata struct {
 	Axis []*Axis `json:"axis"`
 } // @name Metadata
 
+// @Description Fence metadata
+type FenceMetadata struct {
+	// Describes the shape of returned data array.
```

Comment:
"Shape of the returned data fence"? "Describes the" adds nothing, and data array is needlessly unprecise 😄